docs/OCP_CI_Tutorials/Cluster_Cleanup/Manual_Cluster_Cleanup_Guide.md

```diff
@@ -4,13 +4,45 @@
 
 - [Introduction](#introduction)
 - [AWS](#aws)
+  - [General cleanup dos and don'ts](#general-cleanup-dos-and-donts)
+    - [VPCs](#vpcs)
+    - [Hosted zones](#route-53-hosted-zones)
+    - [S3 Buckets](#s3-buckets)
+  - [Cleanup resources by cluster identified](#cleanup-resources-by-cluster-identified)
 
 ## Introduction
 
 Sometimes, a scenario can fail before a cluster is fully deprovisioned leaving stale resources in a cloud-provider (at the time of writing this, we only use AWS). If that occurs, the Interop team is responsible for cleaning that cluster up in the cloud-provider account to avoid any unwanted cost. This document will serve as a guide to how to manually cleanup a cluster in the cloud-provider platforms we use.
```

Comment:
```suggestion
Sometimes, a scenario can fail before a cluster is fully deprovisioned leaving stale resources in a cloud-provider (at the time of writing this, we only use AWS). If that occurs, the Interop team is responsible for cleaning up that cluster in the cloud-provider account to avoid any unwanted cost. This document serves as a guide on how to manually clean up a cluster in the cloud-provider platforms we use.
```